systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -240,4 +240,9 @@
      * Tag for tests where cruise control used
      */
     String CRUISE_CONTROL = "cruisecontrol";
+
+    /**
+     * Tag for tests where mainly dynamic configuration is used
+     */
+    String DYNAMIC_CONFIGURATION = "dynamicconfiguration";
```

Comment:
Is this meant to be only for kafka config or also for the various (kafka, CO, ...) dynamic logging configurations?

Reply:
Everywhere, where the broker will not be triggered by change of dyn. configuration we should tag as `DYNAMIC_CONFIGURATION`